data/kahet/kahet missions.txt

```diff
@@ -721,3 +721,1162 @@ event "pacified ka'sei"
+			choice
+				`	"Won't this anger the Ka'sei?"`
+					goto anger
+				`	"What if you can't access that data?"`
+			`	"That should be the easy part compared to the rest of the mission, so we should just abandon the plan if that happens. Without that, it would be too dangerous to continue; however, it should not be very different from the Ka'het design language in that. After all, the Builders that designed it needed to be able to conduct maintenance on it when the need arose."`
+				goto move
+
+			label anger
+			`	"Almost certainly not, given how negligible the damage should be. A sidearm will not make a scratch in its armor, let alone a dent; and even in the small chance it does activate its defenses, you should have enough time to leave the system and let it calm down if we were to resort to that."`
+			label move
+			`	"Inside, we will search for the memory banks of the spacecraft - large stacks of heavy, faintly glowing cylinders, plain to the outside but containing mechanisms of incredible complexity and precision. After a quick selection based on the data we can scrape, we will take the few that contain information concerning our research and extract their contents. Understood?"`
+			choice
+				`	(Nod.)`
+					goto entrance
+				`	"What about me?"`
+			`	"Given that you need to be able to shoot the target as quick as possible, you cannot return to the cockpit to keep an eye on the scan systems. So, we took the liberty of creating a small terminal here for you to keep an eye situation," he replies, indicating a small desk filled to the brim with Remnant machinery.`
+			label entrance
+			`	One by one, the scientists jump from your ship to the Fetri, moving toward a closed, circular hatch in between the two engines. As you watch them start the expedition, you sit down in the console they have prepared in the cargo bay, monitoring the movements in the system and any change in the slug. Though you can't see clearly what the five of them are doing, you distinguish someone inserting a rod inside a small aperture, followed by the hatch opening right after.`
+			`	As the crew moves beyond your line of sight and the communications cease, your uneasiness about the mission keeps growing steadily. Time passes slowly as you wait for their return; as you split your attention between the various screens in front of you and the occasional glances out of the airlock, every minute seems to last an eternity. And then, it happened.`
+			`	All above the surface of the asteroid-looking creature a burst of rose tinted sparks appears in an instant, quickly growing in number and coverage; a clear sign that it is about to jump away, and that the Fetri'sei is about to awaken. The expedition must end right now, or the Remnant may not survive the encounter.`
```

Comment:
Changed to `	Scanning over the monitors for the umpteenth time, you finally see that heightened energy levels have been detected inside the unknown creature. It is about to jump.` since the passage became to short and unexplanatory